mobify/progressive-web-scaffold
# Change Asset Path

There is an issue in the project scaffold where the `url()` paths defined in CSS files work locally, but don't work in a published bundle.

 **JIRA**: (link to JIRA ticket)
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Move the `/asset` directory into `/app`
- Add the `-url` parameter to the CSS loader

## How to test-drive this PR
- Add the following SCSS to the buttom component's theme:

    ``` scss
    .c-button::after {
        content: url('./static/img/logo.png')
    }
    ```

- `npm run dev`
- Preview on [Merlins Potions](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.min.js&disabled=0&domain=&scope=1)
- Confirm that the image displays correctly inside of the button

## Note
- This change is intended to fix a published bundle issue! I'm not exactly sure the best way to test that for the scaffold... thoughts anyone?
